livefs_edit/actions.py

```diff
@@ -786,3 +793,9 @@ def mount_all_layers(ctxt, target='mounts'):
         squash_names = get_layer_part_names(squash.stem)
         lowers = [ctxt.mount_squash(name) for name in squash_names]
         ctxt.add_overlay(lowers, ctxt.p(f'{target}/{squash.stem}'))
+
+@register_action()
+def add_arch(ctxt, arch=None):
+    if arch is None:
+        raise Exception("argument to --arch is mandatory")
```

Comment:
Do you need to do this? What error do you get if you just remove the default from the `arch` argument and then don't pass it?

Reply:
Good point, the error handling can be removed.